.github/workflows/build.yml

```diff
@@ -8,6 +8,8 @@ jobs:
         uses: actions/checkout@v2
       - name: Install Sphinx
         run: sudo apt-get install python3-sphinx
+      - name: Install Read the Docs theme
+        run: sudo pip install sphinx-rtd-theme
```

Comment:
This isn't needed.  Pipenv will take care of this if we have the theme in our Pipfile and Pipfile.lock.

Reply:
Hey @markpbaggett I did try to remove these and yet it still fails the _build_ workflow run. See https://github.com/UTKcataloging/mods_to_rdf/pull/53/commits/fefbf98d00c355aae83b9f11f093c0cbbfb88a8e